tests/icub-model-test.cpp

```diff
@@ -328,15 +406,44 @@ bool checkFTSensorsAreCorrectlyOriented(iDynTree::KinDynComputations & comp)
                        && checkFTSensorIsCorrectlyOriented(comp, rootLink_R_foot_sensorFrameExpected_plus, "r_foot_ft_sensor");
 
     bool ok = checkFTSensorIsCorrectlyOriented(comp, rootLink_R_sensorFrameExpected, "l_arm_ft_sensor");
-    ok = ok && checkFTSensorIsCorrectlyOriented(comp, rootLink_R_sensorFrameExpected, "r_arm_ft_sensor");
-    ok = ok && checkFTSensorIsCorrectlyOriented(comp, rootLink_R_sensorFrameExpected, "l_leg_ft_sensor");
-    ok = ok && checkFTSensorIsCorrectlyOriented(comp, rootLink_R_sensorFrameExpected, "r_leg_ft_sensor");
-    ok = ok && (checkFTSensorIsCorrectlyOriented(comp, rootLink_R_sensorFrameExpected, "l_foot_ft_sensor") || isPlusModel);
-    ok = ok && (checkFTSensorIsCorrectlyOriented(comp, rootLink_R_sensorFrameExpected, "r_foot_ft_sensor") || isPlusModel);
+    ok = ok & checkFTSensorIsCorrectlyOriented(comp, rootLink_R_sensorFrameExpected, "r_arm_ft_sensor");
+    ok = ok & checkFTSensorIsCorrectlyOriented(comp, rootLink_R_sensorFrameExpected, "l_leg_ft_sensor");
+    ok = ok & checkFTSensorIsCorrectlyOriented(comp, rootLink_R_sensorFrameExpected, "r_leg_ft_sensor");
+    ok = ok & (checkFTSensorIsCorrectlyOriented(comp, rootLink_R_sensorFrameExpected, "l_foot_ft_sensor") || isPlusModel);
+    ok = ok & (checkFTSensorIsCorrectlyOriented(comp, rootLink_R_sensorFrameExpected, "r_foot_ft_sensor") || isPlusModel);
```

Comment:
Not a problem or blocking PR, but please avoid unrelated changes in PRs, especially changes like this that change the use of the logical operator `&&` to the bitwise operator `&` , that in this case should work, but in some cases could lead to unexpected results (see Point 3 of https://www.geeksforgeeks.org/bitwise-operators-in-c-cpp/).

Reply:
I put `&` instead of `&&` in order to check all the fts instead of stopping at the first failure, I can change it  keeping `&&` and inverting the operands

Reply:
> I put `&` instead of `&&` in order to check all the fts instead of stopping at the first failure, I can change it keeping `&&` and inverting the operands

Ahhhh, I am afraid this is not clear at all, and is kind of abusing that `&` is. I suggest to do something like:
~~~
    sensCheck = checkFTSensorIsCorrectlyOriented(comp, rootLink_R_sensorFrameExpected, "r_arm_ft_sensor");
    ok = ok && sensCheck;
    sensCheck = checkFTSensorIsCorrectlyOriented(comp, rootLink_R_sensorFrameExpected, "l_leg_ft_sensor");
    ok = ok && sensCheck;
~~~

Note that if we want this kind of functionalities (i.e. the difference between `REQUIRE` and `CHECK` in catch, see https://github.com/catchorg/Catch2/blob/devel/docs/assertions.md#natural-expressions) probably it could make sense to use for this a  proper testing framework (the aforementioned catch2, for example).